sites/svelte.dev/src/routes/docs/+page.svelte

```diff
@@ -210,7 +210,6 @@
 	}
 
 	onMount(() => {
-		console.log(get_old_new_ids_map());
```

Comment:
Recommend not removing it, its left over for debugging in case a redirect goes wrong